### PR TITLE
MM-876: XSS in relation name (input validation)

### PR DESCRIPTION
**Pull request by jnsereko**
I have encoded the patient name to prevent xss

before:
Screenshot from 2021-07-07 19-07-47

After
Screenshot from 2021-07-07 18-57-11

cc @isears @sherrif10

### CODE REVIEW

File: omod/src/main/webapp/fragments/field/personRelationship.gsp [hunk 1]
```diff
@@ -27,7 +27,7 @@
         </p>
 
         <p class="left">
-            <input type="text" class="person-typeahead" placeholder="${ui.message('registrationapp.person.name')}"
+            <input type="text" class="person-typeahead" placeholder="${ui.message(ui.encodeHtmlContent('registrationapp.person.name'))}"
```
Comment (isears) [inline omod/src/main/webapp/fragments/field/personRelationship.gsp:30]: I think `ui.encodeHtmlAttribute` might be a better fit here

**Review by isears — CHANGES_REQUESTED** [review]

File: omod/src/main/webapp/fragments/field/personRelationship.gsp (hunk 1 above)
Comment (jnsereko) [reply omod/src/main/webapp/fragments/field/personRelationship.gsp:30]: i agree 100%

**Comment by jnsereko** [comment]
@isears i have added some changes
thank you!

**Review by isears — APPROVED** [review]
Great thanks @jnsereko !

**Pull request merged**